src/App.js

```diff
@@ -1,10 +1,206 @@
-import React from 'react';
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import loGo from './assets/screenshot-logo.png';
 import './App.css';
 
 function App() {
+    const [aarbeiCount, setAarbeiCount] = useState(0);
+    const [banaanCount, setBanaanCount] = useState(0);
+    const [appelCount, setAppelCount] = useState(0);
+    const [kiwiCount, setKiwiCount] = useState(0)
+   // const [resetButton, setResetButton] = useState() ??
+
+    const {register, handleSubmit, errors} = useForm({mode:"onChange"});
+
+
+    function onSubmit(data) {
+        console.log(data)
+
+    }
+
+    const resetCount = () => {
+        setAarbeiCount(0);
+        setBanaanCount(0);
+        setAppelCount(0);
+        setKiwiCount(0);
+    }
   return (
     <>
-      <h1>Fruitmand bezorgservice</h1>
+        <div className="container">
+            <img src={loGo} width="295px"/>
+        </div>
+
+      <div className="container">
+    <h3>🍓 Aardbei: </h3>
+
+    <button
+        type="button"
+        name="aarbeiMin"
+        onClick={() => setAarbeiCount(aarbeiCount-1)}
+    >-
+    </button>
+
+    <p>{aarbeiCount}</p>
+
+    <button
+        type="button"
+        name="aarbeiPlus"
+        onClick={() => setAarbeiCount(aarbeiCount+1)}
+    >+
+    </button>
+</div>
```

Comment:
Ik vind de indenting nogal slordig. Ik kan daarom ook niet goed zien hoe de UI opgebouwd is